### ISSUE TITLE
Discussion: Include county-level data?

### ISSUE THREAD

**Issue by nnoll** [neherlab/covid19_scenarios_data#58]
The New York Times has open sourced all their data here:
https://github.com/nytimes/covid-19-data

This includes country level information, which I know we have for Canada already. Up until now we haven't used such granularity in the data. I think we should start to have a discussion for what information/data should we realistically provide to the user of covid19_scenarios? 

I think it may be useful, but only provided there is enough data (threshold on cases?) to integrate it into the app. What do others think?

**Comment by noleti** [comment]
I also wrote a parser for a different project that scrapes county-level data for Germany from RKI (the county-level data from canada I actually  removed it on request). I think the coronadatascraper.com API also gives county-level for other countries (e.g. USA-WA-Kitsap County). coronadatascraper.com actually also provides population data for many regions/counties (not consistently though, I think). But relevant data such as hospital beds and ICU beds would miss for those counties.

**Comment by nnoll** [comment]
Yeah that's my worry as well -- these would have likely have to be extrapolated from the state/province level data which isn't great as ICU and hospitals beds aren't uniformally distributed. It would also lead to a bulky scenario json file

**Comment by noleti** [comment]
I just realized the coronadatascraper.com parser was not correctly pushing results into the `case_counts.json`. I fixed this now, but this will currently lead to county-level data end up in that .json. Should I remove such data/ the whole cds.py for now?

**Comment by nnoll** [comment]
I'm transferring this to the main repo as we are no longer going to keep this as submodule but rather just a directory within covid19_scenarios.
